nix/migration-tests.nix

```diff
@@ -56,7 +56,7 @@ let
       sha256 = "1nfzfdv03nz2bhin2jflxsmh597vkjifb76jsdg0384x89gi0a10"; }
     { rev = "v2020-09-30";
       sha256 = "19vmyq0m67ih295z1y8ng2rdn85mi0rvw26k711lbxsqri7fmyvz"; }
-    { rev = "v2020-10-13";
+    { rev = "v2020.11.3";
```

Comment:
I think this should not change. Migration tests to be updated after release, and the format should be `v2020-11-03` here I suppose.

Reply:
Yes that's right

Reply:
-> https://github.com/input-output-hk/cardano-wallet/pull/2291/commits/1408ac93cb7f3866144f67a9e01e696d4837e03e